cpp/models/abm/location.cpp

```diff
@@ -196,4 +201,73 @@ Eigen::Ref<const Eigen::VectorXi> Location::get_subpopulations() const
     return Eigen::Map<const Eigen::VectorXi>(m_subpopulations.data(), m_subpopulations.size());
 }
 
+void Location::set_default_capacity(LocationType type)
+{
+    if (type == LocationType::Home) {
+        m_capacity.persons = 5;
+        m_capacity.volume  = 104;
+    }
+    else if (type == LocationType::School) {
+        m_capacity.persons = 600;
+        m_capacity.volume  = 150;
+    }
+    else if (type == LocationType::Work) {
+        m_capacity.persons = 10;
+        m_capacity.volume  = 518;
+    }
+    else if (type == LocationType::SocialEvent) {
+        m_capacity.persons = 100;
+        m_capacity.volume  = 150;
+    }
+    else if (type == LocationType::BasicsShop) {
+        m_capacity.persons = 50;
+        m_capacity.volume  = 2000;
+    }
+    else if (type == LocationType::Hospital) {
+        m_capacity.persons = 4;
+        m_capacity.volume  = 100;
+    }
+    else if (type == LocationType::ICU) {
+        m_capacity.persons = 3;
+        m_capacity.volume  = 100;
+    }
+    else if (type == LocationType::Car) {
+        m_capacity.persons = 5;
+        m_capacity.volume  = 4;
+    }
+    else if (type == LocationType::PublicTransport) {
+        m_capacity.persons = 150;
+        m_capacity.volume  = 75;
+    }
+    else {
+        m_capacity.persons = 0;
+        m_capacity.volume  = 0;
+    }
+}
+
+int Location::get_capacity(bool use_volume)
+{
+    if (use_volume) {
+        if (m_capacity.volume > 0) {
+            return m_capacity.volume;
+        }
+        else {
+            mio::log_warning("Volume capacity not set but called. Using capacity in person number.");
+            return m_capacity.persons;
+        }
+    }
+    else {
+        return m_capacity.persons;
+    }
+}
+
+double Location::compute_rel_risk(bool generalized)
```

Comment:
try to avoid abbreviations like `rel` unless they are very well known.

also, more specific name please, `risk` could be anything.